feugy/melodie
# feat(core): adds more audio formats

### What's in there?

Closes #47.

Are included here:

- feat(core): supports opus files
- feat(core): supports wav files
- feat(core): supports webm files
- feat(core): supports weba files

### How to test?

Stop and starts Mélodie: it should find wav/opus/webm/weba files in watched folders.
You can also add  `common/fixtures/` to your watch folders, and search for _fixtures_ and _Test files_ albums.
